nailgun/entities.py

```diff
@@ -3270,6 +3270,68 @@ def search_normalize(self, results):
         return super(Interface, self).search_normalize(results)
 
 
+class JobInvocation(
+        Entity,
+        EntityCreateMixin,
+        EntityReadMixin,
+        EntitySearchMixin):
+    """A representation of a Job invocation entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'description': entity_fields.StringField(),
+            'job_category': entity_fields.StringField(),
+            'targeting_id': entity_fields.IntegerField(),
+            'status': entity_fields.IntegerField(),
+            'start_at': entity_fields.DateTimeField(),
+            'status_label': entity_fields.StringField(),
+            'dynflow_task': entity_fields.ListField(),
```

Comment:
Isn't it ForemanTask? Should be `'dynflow_task': OneToOneField(ForemanTask)` then. And according to your console output it's a dict, not a list anyways.